### ISSUE TITLE
feat(`forge script`): add flag to output `--standard-json-input` somewhere

### ISSUE THREAD

**Issue by bxmmm1** [foundry-rs/book#1350]
### Component

Forge

### Describe the feature you would like

When a script deoloys contracts to 2 different chains (Mainnet, Base) Etherscan & BaseScan don't use the same API key, so the contract verification fails with `--verify`

It would be convenient, to store the standard json output somewhere for all deployed contracts. That way, it is easy to manually drag & drop it on Etherscan,Basescan to verify the contracts.

**Comment by CodeSandwich** [comment]
Awesome! IMO making it a separate command or a feature of a more obvious command like `forge inspect` or `forge build` would be the actual low discoverability solution, but having one more place where it's documented is certainly a good step :+1:

**Comment by bxmmm1** [comment]
Update: Etherscan V2 started using the same API Key for everything 🥇 

https://docs.etherscan.io/etherscan-v2#single-api-key

**Comment by grandizzy** [comment]
> Update: Etherscan V2 started using the same API Key for everything 🥇
> 
Yep, tracking impl in https://github.com/foundry-rs/foundry/issues/9196

**Comment by 2xic** [comment]
_(This might be the wrong place to post it here as this is for the book, but couldn't find a similar ticket in Foundry repo and is related to this issue)_

We also have a use case where we need to get the standard JSON in a more standardized way from Foundry, using the trick [mentioned above](https://github.com/foundry-rs/book/issues/1350#issuecomment-2462197484) won't work for us. Want to contribute the change upstream to help us and others.

I have solved this locally, adding a new command `generate-standard-json` which generates the standard JSON and prints it to stdout. It's done by modularize the logic from the `verify` crate to generate the standard JSON which is then also used by the new command.

Planning on opening a PR in the next few days, but wanted to post here first in case of thoughts / concerns on this.

**Comment by mattsse** [comment]
> Planning on opening a PR in the next few days

awesome, that'd be great

**Comment by CodeSandwich** [comment]
> I have solved this locally, adding a new command `generate-standard-json` which generates the standard JSON and prints it to stdout.

This is very cool, thank you for working on this for all of us! WDYT about making it just another artifact type for `forge inspect`? To me it feels like the right place for the standard JSON, at least this is probably the first place I where I would be looking for it. Just not having another command is a plus too, there are already dozens.

**Comment by CodeSandwich** [comment]
> I agree that forge inspect feels like a more natural home for it. Maybe I'm overthinking it, but the thing I find a bit weird is that forge inspect has a --json format option and having a standard-json field would (in my head) always require the --json format option to be set so those options are a bit in conflict.

I tried `forge inspect` for `metadata` and it printed a JSON both with and without `--json` flag. I think that `standard-json` could behave exactly the same, both of these artifacts are canonically expressed as JSONs.

**Comment by 2xic** [comment]
ah okay, then that sounds like a valid path. Thanks for checking!

**Comment by mattsse** [comment]
marking this as completed https://github.com/foundry-rs/foundry/pull/10537 ty @2xic

**Comment by 2xic** [comment]
> marking this as completed [foundry-rs/foundry#10537](https://github.com/foundry-rs/foundry/pull/10537) ty [@2xic](https://github.com/2xic)

Should I also put up a PR to update the book with the new option, or aren't updates done to the book until next stable release? 

(couldn't find any info on what the update schedule to the book, sorry if I missed it)
